### PR TITLE
Updating SPICE Kernels and adding new Moons to Outer Planets

### PR DESCRIPTION
**Pull request by alexanderbock**
- Updated the kernels for Jupiter, Saturn, Neptune, and Uranus
 - Added new moons from https://ssd.jpl.nasa.gov/sats/ephem

Addresses the kernel part of #2450

### CODE REVIEW

File: data/assets/scene/solarsystem/planets/saturn/minor/norse_group.asset [hunk 1]
```diff
@@ -433,25 +433,24 @@ local norseGroup = {
...
-  --     Path = "/Solar System/Planets/Saturn/Moons/Norse Group"
-  --   },
-  --   TrailColor = trailColor,
-  --   OrbitPeriod = 1242.36,
-  --   Kernels = kernels368
-  -- },
+    Kernels = kernel452
+  },
+  {
+    Identifier = "Surtur", -- Rising
```
Comment (WeirdRubberDuck) [inline data/assets/scene/solarsystem/planets/saturn/minor/norse_group.asset:439]: Is this comment literally just here as a reference to an Amon Amaranth album? ;)

**Review by WeirdRubberDuck — CHANGES_REQUESTED** [review]
Only checked the Neptune moons but hm, I don't think Nereid should look quite like this: 

image

The rendering of the moon trails should also be verified. A lot of the moons end up in the origin of the planet. 
Illustrated by labels here (also, I think the labels should probably live under the title for their respective moons, which they currently seem to not do)
image

**Comment by alexanderbock** [comment]
Addressed the comments;  some of them were copy paste errors, some others were errors on the JPL page that have been reported upstream

**Comment by alexanderbock** [comment]
OBS:
The kernels were changed without updating the version number, so if you have tried it before, you need to remove sync/http/neptune_kernels/2 and redownload.  Or manually add http://liu-se.cdn.openspaceproject.com/files/solarsystem/planets/neptune/kernels/v1_v2/nep095.bsp into that folder


Also for future self:  Whenever we add SPICE items, we should test with the SPICE exceptions enabled as it would have detected those errors

**Pull request merged**